exercises/levenshtein-distance/canonical-data.json

```diff
@@ -0,0 +1,65 @@
+{
+  "exercise": "levenshtein-distance",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Calculate Levenshtein distance",
+      "cases": [
+        {
+          "description": "mine_to_times",
+          "property": "distance",
+          "input": {
+            "phrase": "mine",
+			"phrase": "times"
+          },
+          "expected": "3"
+        },
+		{
+          "description": "_to_abcd",
+          "property": "distance",
+          "input": {
+            "phrase": "",
+			"phrase": "abcd"
```

Comment:
since most JSON readers will only expose one value for each key, you will find it necessary to use different names for the two strings, instead of `phrase` for both. perhaps `from` and `to` will do?